test/sharness/lib/test-lib.sh

```diff
@@ -193,7 +193,7 @@ test_init_ipfs() {
 
   test_expect_success "ipfs init succeeds" '
     export IPFS_PATH="$(pwd)/.ipfs" &&
-    ipfs init --profile=test -b=2048 > /dev/null
+    ipfs init --profile=test -a=rsa -b=2048 > /dev/null
```

Comment:
We should probably switch this from `-a rsa -b=2048` to just nothing. My guess is that `test_init_ipfs` is supposed to be testing the defaults, this means when we switch the defaults in the next PR we'll be affecting all the tests (probably a good thing)